src/test/cypress/e2e/exercises/programming/ProgrammingExerciseAssessment.cy.ts

```diff
@@ -47,6 +47,7 @@ describe('Programming exercise assessment', () => {
                     undefined,
                     undefined,
                     undefined,
+                    undefined,
                     assessmentDueDate,
                     ProgrammingExerciseAssessmentType.SEMI_AUTOMATIC,
```

Comment:
Maybe refactor this method to take an object with named parameters so that this undefined-chain is not necessary.

Reply:
Done :)